pinot-spi/src/main/java/org/apache/pinot/spi/stream/StreamMessageMetadata.java

```diff
@@ -30,23 +30,29 @@
  */
 public class StreamMessageMetadata implements RowMetadata {
   private final long _recordIngestionTimeMs;
+  private final long _recordCreationTimeMs;
```

Comment:
Please add some javadoc on :
1. what this field implies?
2. Since this is part of the stream spi, how is a new stream connector expected to set this field?

Reply:
I have renamed to be more clear, and adding comments. The way to set this field is similar to the one in line 32: via the constructor: strings that do support the field provide as an argument when creating the stream message